infrastructure/modules.tf

```diff
@@ -30,3 +30,20 @@ module "db" {
   common_tags        = var.common_tags
   subscription       = var.subscription
 }
+
+module "db-v11" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=postgresql_tf"
+  product            = var.product
+  component          = var.component
+  name               = join("-", [var.product,var.component,"postgres-db-v11"])
+  location           = var.location
+  env                = var.env
+  postgresql_user    = "ethos"
+  database_name      = "ethos"
+  postgresql_version = "11"
+  subnet_id          = data.azurerm_subnet.postgres.id
```

Comment:
data.azurerm_subnet.postgres needs adding back in otherwise this will cause terraform to fail

Reply:
Refactored